ansible/roles/common/tasks/dependencies.yml

```diff
@@ -15,6 +15,7 @@
       - imagemagick
       - jq
       - libatlas-base-dev
+      - libgfortran5
```

Comment:
You have added this here but not deleted the stretch only block below like I did in #491

Reply:
agree - i did that manually and forgot to remove stretch